website/scripts/populate.js

```diff
@@ -89,11 +89,21 @@ function createFactCard(item) {
     newFactInner.classList.add("factcard");
     let factQuote = document.createElement("blockquote");
     factQuote.classList.add("factquote");
-    factQuote.innerHTML = item['fact'];
+    factQuote.innerHTML = escape(item['fact']);
     let factAttribution = document.createElement("p");
-    factAttribution.innerHTML = "-- " + item['source'];
+    factAttribution.innerHTML = "-- " + escape(item['source']);
     newFactInner.appendChild(factQuote);
     newFactInner.appendChild(factAttribution);
     newFact.appendChild(newFactInner);
     topLevel.appendChild(newFact);
 }
+
+function escape(s) {
+    let lookup = {
+        '&': "&amp;",
```

Comment:
OWASP recommends `'` and `/` too: https://cheatsheetseries.owasp.org/cheatsheets/Cross_Site_Scripting_Prevention_Cheat_Sheet.html#rule-1---html-escape-before-inserting-untrusted-data-into-html-element-content

Reply:
yEET